sacn/messages/root_layer.py

```diff
@@ -53,15 +55,29 @@ def length(self, value: int):
         self._length = value & 0xFFF  # only use the least 12-Bit
 
 
-def int_to_bytes(integer: int) -> list:
+def int_to_bytes(integer_value: int) -> list:
     """
     Converts a single integer number to an list with the length 2 with highest
     byte first.
     The returned list contains values in the range [0-255]
     :param integer: the integer to convert
     :return: the list with the high byte first
     """
-    return [(integer >> 8) & 0xFF, integer & 0xFF]
+    if not (isinstance(integer_value, int) and 0 <= integer_value <= 65535):
+        raise TypeError(f'integer_value to be packed must be unsigned short: [0-65535]! value was {integer_value}')
+    return list(struct.pack('!H', integer_value))
```

Comment:
I'd originally gone there because it's written to sort out everything about host endianness and any other details I wasn't consider.  I've reworked to make byte_tuple_to_int just do the converse bit-shift operations from int_to_bytes.